tools/cfn2ts/lib/codegen.ts

```diff
@@ -702,3 +725,33 @@ function tagType(resource: schema.ResourceType): string {
   }
   return `${TAG_TYPE}.NotTaggable`;
 }
+
+enum Container {
```

Comment:
I don't love my naming here, what is the right CS term here? Scope? but we use scope in another context.